packages/loader/container-loader/src/container.ts

```diff
@@ -592,6 +593,13 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                     [DriverHeader.createNew]: {},
                 };
             }
+            // Add container package name to the request url.
+            const codeDetails = this.context.codeDetails;
+            const packageName = isFluidPackage(codeDetails.package) ? codeDetails.package.name : codeDetails.package;
+            const [mainUrl, queryString] = request.url.split("?");
+            const searchParams = new URLSearchParams(queryString);
+            searchParams.append("containerPackageName", packageName);
+            request.url = `${mainUrl}?${searchParams.toString()}`;
```

Comment:
none of this code should be here. the container shouldn't have any coupling to any url formats

Reply:
Removed.